### ISSUE TITLE
electron dumps core

### ISSUE THREAD

**Issue by tpapp** [JuliaGizmos/Blink.jl#184]
```julia
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.2.0-DEV.109 (2019-01-06)
 _/ |\__'_|_|_|\__'_|  |  Commit f8f20453c4 (0 days old master)
|__/                   |

(v1) pkg> st Blink
    Status `~/.julia/environments/v1/Project.toml`
  [ad839575] Blink v0.9.0

julia> using Blink

julia> run(`$(Blink.AtomShell.electron())`)
ERROR: failed process: Process(`/home/tamas/.julia/packages/Blink/6QR5U/deps/atom/electron`, ProcessSignaled(11)) [0]
Stacktrace:
 [1] error(::String, ::Base.Process, ::String, ::Int64, ::String) at ./error.jl:42
 [2] pipeline_error at ./process.jl:785 [inlined]
 [3] #run#516(::Bool, ::Function, ::Cmd) at ./process.jl:726
 [4] run(::Cmd) at ./process.jl:724
 [5] top-level scope at none:0
```

First I thought it was #128, but narrowed it down to electron not starting following the advice there.

Ubuntu 18.10, happy to provide more information for debugging.

**Comment by tpapp** [comment]
Tried
```julia
julia> Blink.AtomShell.uninstall()

julia> Blink.AtomShell.install()
```
does not help. I wonder if I could somehow install electron via nodejs and tell Blink about it?

**Comment by piever** [comment]
I'm having similar problems, always on Ubuntu 18.10 (and Blink 0.9).

I can reproduce the above error and if If I try to open a window with `Window()` I get:

```julia
julia> using Blink

julia> Window()
ERROR: IOError: connect: connection refused (ECONNREFUSED)
Stacktrace:
 [1] try_yieldto(::typeof(Base.ensure_rescheduled), ::Base.RefValue{Task}) at ./event.jl:196
 [2] wait() at ./event.jl:255
 [3] wait(::Condition) at ./event.jl:46
 [4] stream_wait(::Sockets.TCPSocket, ::Condition) at ./stream.jl:47
 [5] wait_connected(::Sockets.TCPSocket) at ./stream.jl:330
 [6] connect(::Sockets.TCPSocket, ::Sockets.IPv4, ::Int64) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.1/Sockets/src/Sockets.jl:456
 [7] connect(::Sockets.IPv4, ::Int64) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.1/Sockets/src/Sockets.jl:440
 [8] #try_connect#5(::Float64, ::Int64, ::Function, ::Sockets.IPv4, ::Vararg{Any,N} where N) at /home/pietro/.julia/packages/Blink/6QR5U/src/AtomShell/process.jl:71
 [9] try_connect at /home/pietro/.julia/packages/Blink/6QR5U/src/AtomShell/process.jl:69 [inlined]
 [10] #init#6(::Bool, ::Function) at /home/pietro/.julia/packages/Blink/6QR5U/src/AtomShell/process.jl:85
 [11] #init at ./none:0 [inlined]
 [12] #shell#9(::Bool, ::Function) at /home/pietro/.julia/packages/Blink/6QR5U/src/AtomShell/process.jl:123
 [13] shell at /home/pietro/.julia/packages/Blink/6QR5U/src/AtomShell/process.jl:121 [inlined]
 [14] #Window#12(::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Type) at /home/pietro/.julia/packages/Blink/6QR5U/src/AtomShell/window.jl:70
 [15] Window() at /home/pietro/.julia/packages/Blink/6QR5U/src/AtomShell/window.jl:70
 [16] top-level scope at none:0
```

**Comment by piever** [comment]
The following hack works:

```
npm install electron -g

julia> Blink.AtomShell.electron() = "electron"
```

I wonder if it is simply an issue with the version installed by Blink being incompatible with Ubuntu 18.10?

**Comment by travigd** [comment]
Fixed(?) by #187.

Feel free to open a new issue if you're still having problems.
